object.go

```diff
@@ -84,6 +84,13 @@ func (o *Object) SetIdx(idx uint32, val interface{}) error {
 	return nil
 }
 
+// Returns an array containing the names of the enumerable properties of this object,
+// including properties from prototype objects
+func (o *Object) GetPropertyNames() []string {
```

Comment:
`v8::Object::GetPropertyNames()` returns `MaybeLocal<Array>`, which means that it can fail and that it returns a JS array object instead of a native array for the host.  So I would expect the return value to be `(*Array, error)` in v8go to better match the underlying v8 method, although it looks like we don't yet have an Array type.  The array contents can be retrieved using Object.GetIdx as mentioned in the corresponding issue for this PR (https://github.com/rogchap/v8go/issues/213).

V8 also overloads this method with a more generic version with more control over filtering (this would only return enumerable properties) and key conversion (this would convert indexes to strings).  This more generic version of the method seems like it would be the more appropriate one to expose with this generic `GetPropertyNames` method name.

Given that this is basically exposing a convenience method for returning enumerable properties, perhaps it should be named `GetEnumerablePropertyNames` to better reflect what it returns.  Once viewed in this way as a convenience method, having `([]string, error)` be the return value seems appropriate, since getting a Go slice would be more convenient for typical use and this type would reflect the fact that indexes get converted to strings.

```suggestion
func (o *Object) GetEnumerablePropertyNames() ([]string, error) {
```

Reply:
Fixed here https://github.com/rogchap/v8go/pull/260/commits/19f8b5c1d27a23166f28c79bf792806a5990dc67